src/imageUtil.ml

```diff
@@ -212,6 +212,15 @@ let print_byte v =
  *)
 let rec ones i = if i == 1 then 1 else ((ones (i-1)) lsl 1) lor 1 ;;
 
+(*
+ * Converts a string of size 2 into an integer
+ * Arguments:
+ *   - s : the string.
+ * Returns an integer.
+ *)
+let int_of_str2_le s =
+  (int_of_char s.[0]) lor ((int_of_char s.[1]) lsl 8)
+
 (*
  * Converts a string of size 4 into an integer WITHOUT taking care of
```

Comment:
@rlepigre should we add a comment here detailing that it's a big-endian decoder?
Also, how is this intended to work on 32bit platforms?

Reply:
@cfcs: Well, this part will probably need to use `int32` at some point, but this is going to be annoying. I wish OCaml did not need to have 31 (or 63) bits integers...